src/app/games/tafl/tests/MyTaflMove.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { TaflMove } from '../TaflMove';
+
+export class MyTaflMove extends TaflMove {
```

Comment:
why define move & state in separate files, but config & rules in Tafl.spec ?

Reply:
Parce que MyTaflMove et MyTaflState sont importés, mais pas MyConfig ?

Reply:
yé kroi